AdyenActions/Components/Voucher/VoucherComponentExtensions.swift

```diff
@@ -86,8 +86,6 @@ extension VoucherComponent: VoucherViewDelegate, DocumentActionViewDelegate {
     }
     
     private func createSaveAlertAction(for action: VoucherAction, sourceView: UIView) -> UIAlertAction? {
-        guard canAddPasses(action: action.anyAction) else { return nil }
```

Comment:
We now can remove optionality for return type.

```suggestion
private func createSaveAlertAction(for action: VoucherAction, sourceView: UIView) -> UIAlertAction? {
```